yaksh/templates/yaksh/question.html

```diff
@@ -155,13 +155,25 @@
                       <h4>
                         <u> {{ question.summary }}
                              {% if question.type == "mcq" %}
-                             (Single Correct Choice Questions)
+                             (SINGLE CORRECT CHOICE)
                              {% elif question.type == "mcc" %}
-                             (Multiple Correct Choices)
+                             (MULTIPLE CORRECT CHOICES)
                              {% elif question.type == "code" %}
                              (PROGRAMMING)
                              {% elif question.type == "upload" %}
                              (ASSIGNMENT UPLOAD)
+                             {% elif question.type == "integer" %}
+                             (ANSWER IN INTEGER)
+                             {% elif question.type == "string" %}
+                             (ANSWER IN STRING)
+                                {% if testcase.string_check == "lower" %}
+                                    <h5>(CASE INSENSITIVE)</h5>
+                                {% else %}
+                                    <h5>(CASE SENSITIVE)</h5>
+                                {% endif %}
+
+                             {% elif question.type == "float" %}
+                             (ANSWER IN FLOAT)
```

Comment:
Just a suggestion. Instead of giving name as `Answer in Float`, Can't we write `Fill in the blanks(Float or Float value)`?. Same in above cases also.